### DISCUSSION TITLE
EJBCA UI cannot be accessed

### DISCUSSION THREAD

**Discussion by XYQ202** [Keyfactor/ejbca-ce#63] [Q&A]
I installed EJBCA correctly on Ubuntu according to the **quick installation guide**, and I can also access the CA UI and other interfaces, but when I close ubuntu, I can't access the CA UI again, and it consistently shows that I can't connect, but wildfly is running Yes, this is a problem, how can I solve it. That is, when the system is shut down, it can still be accessed next time because the p12 certificate has been imported.

**Comment by XYQ202** [comment]
Your team's support isn't great.

**Reply by primetomas** [reply]
It's been three hours? This is a best effort Community support channel. It is vacation period in parts of Europe now, so there are fewer people available to help.
If you need commercial support with SLA this is available from Keyfactor. See https://www.ejbca.org/support/.

**Reply by primetomas** [reply]
There are usually no problems restarting EJBCA it works in production at thousands of places. Something that may happen is that there is something on your system, perhaps the database?, that does not start. 
You should look in the wildfly server.log, it should give clear indications. You are free to post a server.log from a restart that does not work here. The same information as mentioned in #57, #58, #59 applies, where you can find the wildfly server.log file.

Cheers, and have a great summer (pardon slow responses, while we are on vacation).
